examples/tornjak/values.yaml

```diff
@@ -4,6 +4,7 @@ spire-server:
 
 tornjak-frontend:
   enabled: true
+  apiServerURL: "http://localhost:10000"
```

Comment:
```suggestion
  serverURL: "http://localhost:10000"
```

Reply:
This is for the backend url not auth

Reply:
Are you suggesting to change that to serverURL instead of apiServerURL?

Reply:
doh. I misread. its apiServerURL not authServerURL. Sorry for the confusion.